subworkflows/nf-core/utils_nfcore_pipeline/meta.yml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "UTILS_NFCORE_PIPELINE"
+description: Subworkflow with utility functions specific to the nf-core pipeline template
+keywords:
+  - utility
+  - pipeline
+  - initialise
+  - version
+components: []
+input:
+  - nextflow_cli_args:
+      type: list
+      description: |
+        Nextflow CLI positional arguments
+output:
+  - success:
+      type: boolean
+      description: |
+        Dummy output to indicate success
+authors:
+  - "@adamrtalbot"
```

Comment:
You don't think you should be author?

Reply:
Hmm, these are pulled as is from nf-core, so seems reasonable that those who created/maintain them are mentioned.